flow360/component/simulation/models/validation/validation_bet_disk.py

```diff
@@ -0,0 +1,133 @@
+"""
+validation BETDisk
+"""
+
+
+# pylint: disable=missing-function-docstring
+def _get_bet_disk_name(bet_disk):
```

Comment:
Can we generalize this so it can be shared?
I can see boundary condition and output requiring same treatment.
please add additional argument for getting class name. Put the function temporarily at 
flow360/simulation/validation_utils.py

Reply:
what do you mean by getting class name? could you give some code?

Reply:
"BET disk" is the "class name"?
It is not strictly a "class" name since you cannot just use `bet_disk.__class__.__name__`.

Reply:
I think it may be better to write a wrapper to append this name to the exception, this is more concise.

Reply:
let me try.